app/views/assessment/AssessmentCaptcha.js

```diff
@@ -0,0 +1,142 @@
+import AssessmentButton from './AssessmentButton';
+import { AnswersContext, SurveyContext } from './AssessmentContext';
+
+const WIDTH = Dimensions.get('window').width;
+const HEIGHT = WIDTH * (300 / 375);
+
+/** @type {React.FunctionComponent<{}>} */
+const AssessmentCaptcha = ({ navigation }) => {
+  let { t } = useTranslation();
+  const survey = useContext(SurveyContext);
+  const answers = useContext(AnswersContext);
+  const [token, setToken] = useState(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const buttonDisabled = token === null || isLoading;
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView style={styles.scrollView}>
+        <Image
+          reiszeMode='cover'
+          source={require('../../assets/images/illustration-screening-data-sharing.png')}
+          style={{ width: WIDTH, height: HEIGHT }}
+        />
+        <Text style={styles.title}>{t('assessment.captcha_title')}</Text>
+        <Text style={styles.description}>
+          {t('assessment.captcha_description')}
+        </Text>
+
+        <HCaptcha
+          style={styles.captcha}
+          uri={CATCHA_URL}
+          onMessage={event => {
+            if (event && event.nativeEvent.data) {
+              if (
+                ['cancel', 'error', 'expired'].includes(event.nativeEvent.data)
+              ) {
+                console.log('Verified code failed');
+                Alert.alert(
+                  'Captcha Response',
+                  `captcha ${event.nativeEvent.data}`,
+                  [
+                    {
+                      text: 'OK',
+                    },
+                  ],
+                );
+              } else {
+                console.log('Verified code received', event.nativeEvent.data);
+                setToken(event.nativeEvent.data);
+              }
+            }
+          }}
+        />
+      </ScrollView>
+      <View style={styles.footer}>
+        <AssessmentButton
+          disabled={buttonDisabled}
+          onPress={() => {
+            setIsLoading(true);
+            const state = navigation.dangerouslyGetState();
+            const questionKeys = survey.questions.map(q => q.question_key);
+            // Extract the keys from navigation stack because the user might
+            // have answered a question, pressed back, changed answer, ended up on a different question
+            // rendering that answer no longer valid
+            const questionKeysFinal = state.routes
+              .filter(r => r.params && r.params.question)
+              .map(r => r.params.question.question_key)
+              // Remove injected questions (agreement, etc)
+              .filter(k => questionKeys.includes(k));
+            const response = questionKeysFinal.map(question_key => ({
+              question_key,
+              response: answers[question_key].map(r => r.value),
+            }));
+            // TODO: Loading state / disable button
+            axios
```

Comment:
This is the only place you use axios. and it seems the usage is simple enough to warrant just using `fetch()` -- there is no complex response parsing, there is no complex body encoding. 

Is axios really needed?